dashboard/lib/types/top-sources.ts

```diff
@@ -1,10 +1,11 @@
-export type TopSourcesData = {
-  referrer: string
-  visits: number
-}
-
 export type TopSource = {
   referrer: string
   visits: number
   href?: string
 }
+
+export type TopSources = {
+  data: Array<TopSource>
```

Comment:
really silly change, just to be consistent

```suggestion
  data: TopSource[]
```